imaging_tracking/settings.py

```diff
@@ -127,20 +127,11 @@
 
 # Auth redirects
 # https://docs.djangoproject.com/en/3.0/ref/settings/#std:setting-LOGIN_REDIRECT_URL
-LOGIN_REDIRECT_URL = '/import/'
+LOGIN_REDIRECT_URL = '/'
 LOGOUT_REDIRECT_URL = '/accounts/login/'
 
-# LDAP authentication backend
-# https://django-auth-ldap.readthedocs.io/
-AUTH_LDAP_SERVER_URI = "ldap://ldap-ro.internal.sanger.ac.uk"
-AUTH_LDAP_USER_DN_TEMPLATE = "uid=%(user)s,ou=people,dc=sanger,dc=ac,dc=uk"
-AUTH_LDAP_USER_ATTR_MAP = {
-    "username":     "uid",
-    "first_name":   "sangerGivenNameOverride",
-    "last_name":    "sangerSnOverride",
-    "email":        "mail",
-}
-AUTHENTICATION_BACKENDS = [
-    'django_auth_ldap.backend.LDAPBackend',
-    'django.contrib.auth.backends.ModelBackend',
-]
\ No newline at end of file
+# Email
+EMAIL_BACKEND = "django.core.mail.backends.smtp.EmailBackend"
+EMAIL_HOST = "mail.sanger.ac.uk"
+EMAIL_PORT = 25
+DEFAULT_FROM_EMAIL = "no-reply@cellgeni.sanger.ac.uk"
```

Comment:
still sent an email from sanger.ac.uk . We can leave it "sanger.ac.uk" with a "TODO" to change it to something less global and also with a note that "cellgeni.sanger.ac.uk" was still sending emails from sanger.ac.uk

Reply:
Quick tests show that any invalid email, not complaing with rfc5322?, ie `address@domain` or `address@` gets send as `address[@sanger.ac.uk]`, also, sanger.ac.uk does not allow sending emails with subdomains (cellgeni.sanger.ac.uk)

Reply:
should it be something like `cellgeni-no-reply@sanger.ac.uk` then?

Reply:
Yes. That or simply a fantasy email like `no-reply@cellgeni.email`